x-pack/plugins/apm/public/components/app/backend_detail_overview/backend_detail_dependencies_table.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { getNodeName, NodeType } from '../../../../common/connections';
+import { useApmParams } from '../../../hooks/use_apm_params';
+import { useApmRouter } from '../../../hooks/use_apm_router';
+import { useUrlParams } from '../../../context/url_params_context/use_url_params';
+import { useFetcher } from '../../../hooks/use_fetcher';
+import { getTimeRangeComparison } from '../../shared/time_comparison/get_time_range_comparison';
+import { DependenciesTable } from '../../shared/dependencies_table';
+import { NodeIcon } from '../../shared/node_icon';
+import { useApmBackendContext } from '../../../context/apm_backend/use_apm_backend_context';
+
+export function BackendDetailDependenciesTable() {
+  const {
+    urlParams: { start, end, environment, comparisonEnabled, comparisonType },
+  } = useUrlParams();
```

Comment:
I know we still do magic with `useUrlParams` and start/end, but can the rest fo them come out of the `useApmParams` below?

Reply:
I think the others have slightly different types/defaults as well. I'm trying to err on the safe side here, and then hopefully we can refactor these separately.

Reply:
Do we have an issue to refactor this?

Reply:
We have https://github.com/elastic/kibana/issues/105914